tests/core/api/GH_ORG_SUBS_API.js

```diff
@@ -0,0 +1,436 @@
+'use strict';
```

Comment:
the only issue i have here is the name of the file. its not GH_ORG...its just subscriptions.js right? The api does not give you different results when you use different source control. by qualifying this by GH_ORG you are hinting that we should do a bitbucket test also which is pointless.. All we are testing here is the subs and making sure the subs are returned correctly.